WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -26,21 +39,55 @@ final class BlazeCampaignDashboardViewHostingController: SelfSizingHostingContro
     }
 }
 
+private extension BlazeCampaignDashboardViewHostingController {
+    /// Handles navigation to the campaign creation web view
+    func navigateToCampaignCreation(source: BlazeSource, productID: Int64? = nil) {
+        guard let site = ServiceLocator.stores.sessionManager.defaultSite else {
```

Comment:
Nit: Consider injecting the site into `BlazeCampaignDashboardViewModel` and using it instead of accessing the `ServiceLocator`

Reply:
Since the injection of the site to the view model is quite complicated, I'll add it in a separate PR and merge this for now.